### DISCUSSION TITLE
How to close only one preview and how to open focused preview in current split but new buffer?

### DISCUSSION THREAD

**Discussion by alloc33** [rmagatti/goto-preview#86] [Q&A] [answered]

**Comment by rmagatti** [comment, marked as answer]
@xbladesub I have converted your issue into a discussion instead.

> How to close only one preview

The preview is just a regular Neovim window, you can close it like you would any normal window or split. `:q`.

> how to open focused preview in current split but new buffer

Since the preview window is just a regular Neovim window, you can manipulate it as such. Meaning you can use window movements. `:h window-moving`

**Reply by alloc33** [reply]
Thanks for answer! I expected that preview will close with `<C-c>`, probably need to add some special keymap

**Comment by kamalarieff** [comment]
@rmagatti in your demo animation gif, what keybind did you use to make the floating window into a vertical split and closing the preview floating window?

When I tried to use `C-w v`, it only created the vertical split but the floating window didn't close.

**Reply by rmagatti** [reply]
`<C-w>l` It's just a normal Neovim window movement. 

You can refer to the help for more details on those
`:h CTRL-W`

**Reply by kamalarieff** [reply]
When I press `<C-w>l`, it just goes back to the window that I invoked the command and doesn't close the window. ~~Maybe some other plugin is interfering.~~ Thanks for your help.

I've tried deactivating ~~`dressing.nvim` and `nui.nvim`~~ all plugins but no luck.

**Reply by rmagatti** [reply]
Oh sorry, I meant to say `<C-w>L` you need the capital `L`. 

You should probably read the help around window movements:
`:h CTRL-W_L`

**Reply by kamalarieff** [reply]
Thank you! This one works :+1:
